src/libraries/System.Data.Common/src/System/Data/SQLTypes/SQLGuid.cs

```diff
@@ -328,6 +318,11 @@ public static XmlQualifiedName GetXsdType(XmlSchemaSet schemaSet)
             return new XmlQualifiedName("string", XmlSchema.Namespace);
         }
 
-        public static readonly SqlGuid Null = new SqlGuid(true);
+        void ISerializable.GetObjectData(SerializationInfo si, StreamingContext sc)
```

Comment:
Nit: Since this is an explicit interface implementation, consider keeping the parameter names the same as they are defined on the interface.

```cs
GetObjectData(SerializationInfo info, StreamingContext context);
```

The names of the corresponding ctor parameters don't matter that much, but this interface method is part of the API contract so should match.